css/dashboard.css

```diff
@@ -91,7 +91,9 @@ fieldset.crm-inactive-dashlet-fieldset legend {
 #civicrm-dashboard .ui-sortable-placeholder {
   border: 2px dashed grey;
   visibility: visible !important;
-  height: 80px !important;
+  width: 226px;
```

Comment:
What's the significance of the number 226? This layout is fluid using flex css, so there aren't any fixed widths to speak of.

Reply:
The draggable items are 230px wide, so this sets the correct width of the placeholder, i.e. 230px - 4px border.

Reply:
Hence the 66px height too